synchronization/futex_test.rs

```diff
@@ -0,0 +1,27 @@
+use futex;
+
```

Comment:
Do you think looking for a test framework to use would be helpful?

Reply:
No rust's built-in test stuff is going to be fine, I think